internal/index/scip.go

```diff
@@ -61,13 +62,15 @@ func ListMissing(opts config.IndexOpts) (missing []string, err error) {
 		return nil, err
 	}
 
-	lookupNames := funk.Keys(pkgLookup)
+	lookupNames := maps.Keys(pkgLookup)
+	slices.Sort(lookupNames)
```

Comment:
nit: Maybe rename `funk.Keys` to `funk.SortedKeys` with this implementation and use that here? Maybe even just inline the `SortedKeys` call into the `range` expression instead of creating a temporary which isn't used elsewhere.

Reply:
Thats something i wanted to raise actually, i dont see why we sort anything here at all outside of perhaps some level of determinism